Code snippets/src/main/java/use_case/instructions/InstructionsInteractor.java

```diff
@@ -6,9 +6,9 @@
 
 public class InstructionsInteractor implements InstructionsInputBoundary{
 
-    private FileInstructionsDataAccessObject fileInstructionsDataAccessObject;
+    private InstructionsUserDataAccessInterface fileInstructionsDataAccessObject;
```

Comment:
Good to be more generic here to specify the type as an interface—more extensibility (we can substitute in any other object that implements the InstructionUserDataAccessInterface